app/Helpers/Helpers.php

```diff
@@ -297,6 +297,69 @@ function validateV2Annotation($annotation, $filesets, $books, $v4_users, $v4_ann
     }
 }
 
+if (!function_exists('validateV2Annotation')) {
```

Comment:
this function already existed, didn't it? you added the boolean at the end. please revert to the original code since there is now a separate validate for livebibleis